fetch.bs

```diff
@@ -2133,6 +2133,18 @@ Unless stated otherwise, it is false.
 
 <p class=note>This flag is for exclusive use by HTML's render-blocking mechanism. [[!HTML]]
 
+<p>A <a for=/>request</a> has an associated boolean <dfn export for=request>is HTTPS upgrade</dfn>.
+Unless stated otherwise, it is false.
+
+<p class=note>This flag is for exclusive use by HTTPS Upgrading algorithm.
+
+<p>A <a for=/>request</a> has an associated <dfn export for=request>HTTPS upgrade fallback URL</dfn>
+which is a <a for=/>URL</a>. It is a copy of the <a for=/>request</a>'s original
+<a for=request>URL</a> before <a for=/>request</a> is <a href="#https-upgrades-upgrade">optionally
+upgraded to HTTPS</a>. Unless stated otherwise, it is unset.
```

Comment:
This is weird. I think it should be null or a URL and by default it's null all as the algorithm below seems to explicitly initialize it.

Reply:
Done.